web/frontend/src/components/utils/CloseButton.tsx

```diff
@@ -2,15 +2,16 @@ import { AuthContext } from 'index';
 import { useContext } from 'react';
 import { useTranslation } from 'react-i18next';
 import { STATUS } from 'types/electionInfo';
+import { Role } from 'types/userRole';
 
 const CloseButton = ({ status, handleClose }) => {
   const authCtx = useContext(AuthContext);
   const { t } = useTranslation();
 
-  const isAuthorized = authCtx.role === 'admin' || authCtx.role === 'operator';
+  const isAuthorized = authCtx.role === Role.Admin || authCtx.role === Role.Operator;
```

Comment:
For consistency with STATUS and the rest of the CONSTS/ENUMS shouldn't Role be in upper case `ROLE` ?

Reply:
Right !